programs/beautysh.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, pkgs, config, ... }:
+let
+  cfg = config.programs.beautysh;
+in
+{
+  options.programs.beautysh = {
+    enable = lib.mkEnableOption "beautysh";
+    package = lib.mkPackageOption pkgs "beautysh" { };
+  };
+
+  config = lib.mkIf cfg.enable {
+    settings.formatter.beautysh = {
+      command = cfg.package;
+      options = [ "-i" "4" ];
```

Comment:
We're using an indent of 2 in `shfmt`. I think we should either switch this one to 2 or the other one to 4. Maybe also add an `indent` option to make this configurable.

Reply:
Thanks for the advice,I add the indent_size option